### PR TITLE
Update LP tests to run on OCP-4.17 

### PR DESCRIPTION
**Pull request by aviavraham**

### CODE REVIEW

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 1]
```diff
@@ -589,3 +589,199 @@ periodics:
...
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.17-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.17-lp-interop-eap-74-xp-interop-aws
+  slack:
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml]:
```suggestion
  reporter_config:
     slack:
```

**Comment by aviavraham** [comment]
@dfrazzette I suspect that running "make jobs"  multiple times changes some of the files
I will review all the periodic configuration files again and push an update to the PR.

**Pull request merged**